styleguide/derek/_partials/_basement.ejs

```diff
@@ -12,7 +12,7 @@
       </div>
       <div class="col-sm-3 col-sm-push-6">
         <p class="uppercase copyright basement-copyright">&reg; 2016 Rackspace US, Inc.</p>
-        <div id="teconsent" style="display:none;"></div>
+        <div id="teconsent""></div>
```

Comment:
Not sure if `style="display: none;"` is needed, but if not we still have double quotes here that need to be fixed.